packages/atomic/src/components/search/result-template-components/atomic-result-link/atomic-result-link.tsx

```diff
@@ -44,7 +44,7 @@ export class AtomicResultLink implements InitializableComponent {
    *
    * The template literal can reference any number of result properties from the parent result. It can also reference the window object.
    *
-   * For example, the following markup generates an `href` value such as `http://uri.com?id=itemTitle`.
+   * For example, the following markup generates an `href` value such as `http://uri.com?id=itemTitle`, using the item `clickUri` and `itemtitle` fields.
```

Comment:
I'm worried somebody could read this and misunderstand, thinking that we have a separate concept from results called "items".

```suggestion
   * For example, the following markup generates an `href` value such as `http://uri.com?id=itemTitle`, using the result's `clickUri` and `itemtitle` fields.
```